Riot/Modules/TabBar/MasterTabBarController.m

```diff
@@ -979,10 +981,13 @@ - (void)presentVerifyCurrentSessionAlertIfNeededWithSession:(MXSession*)session
     
     self.reviewSessionAlertHasBeenDisplayed = YES;
     [self presentVerifyCurrentSessionAlertWithSession:session];
+#endif
 }
 
 - (void)presentVerifyCurrentSessionAlertWithSession:(MXSession*)session
 {
+// Tchap: Disable Cross Signing Managment.
+#ifdef SUPPORT_KEYS_BACKUP
```

Comment:
Please uncomment "[[AppDelegate theDelegate] presentCompleteSecurityForSession:session];"

Reply:
Fixed in last commit